pydrive2/files.py

```diff
@@ -275,7 +278,9 @@ def SetContentFile(self, filename):
     :param filename: name of the file to be uploaded.
     :type filename: str.
     """
-        self.content = open(filename, "rb")
+        if os.path.getsize(filename):
+            self.content = open(filename, "rb")
```

Comment:
Also not sure if self.content is ever reused, since we just not set it here and don't reset it.

Reply:
It is really not meant to be reset: https://github.com/iterative/PyDrive2/blob/18ec4cfb86d7272e224c6bd42778ef5af1c1eb5b/pydrive2/apiattr.py#L4